cmd/kubeadm/app/phases/etcd/OWNERS

```diff
@@ -0,0 +1,2 @@
+labels:
+  - sig/etcd
```

Comment:
we can omit the SIG etcd label from cmd/kubeadm.
these are strictly kubeadm bits that we own, but we can certainly ask for review expertise if needed.

Reply:
Done!